backend/src/test/java/heartbeat/controller/board/JiraControllerTest.java

```diff
@@ -59,27 +60,31 @@ void shouldHandleServiceExceptionAndReturnWithStatusAndMessage() throws Exceptio
 		when(mockException.getStatus()).thenReturn(400);
 
 		BoardRequestParam boardRequestParam = BOARD_REQUEST_BUILDER().build();
-		MultiValueMap<String, String> parameters = buildParameter(boardRequestParam);
 
-		mockMvc.perform(get("/boards/{boardType}", "jira").params(parameters))
+		mockMvc
+			.perform(post("/boards/{boardType}", "jira").contentType(MediaType.APPLICATION_JSON)
+				.content(new ObjectMapper().writeValueAsString(boardRequestParam)))
 			.andExpect(status().isBadRequest())
 			.andExpect(jsonPath("$.message").value(message));
 	}
 
 	@Test
 	void shouldVerifyRequestTokenNotBlank() throws Exception {
-		BoardRequestParam boardRequestParam = BOARD_REQUEST_BUILDER().token("").build();
-		MultiValueMap<String, String> parameters = buildParameter(boardRequestParam);
+		BoardRequestParam boardRequestParam = BOARD_REQUEST_BUILDER().token(null).build();
```

Comment:
your test case is `shouldVerifyRequestTokenNotBlank()`, but you pass `null` into token, make sense???

Reply:
and update this test case name to `shouldABadRequestWhenTokenIsBlank()`